packages/devextreme/js/__internal/ui/list/m_list.base.ts

```diff
@@ -555,28 +561,45 @@ export class ListBase extends CollectionWidget<ListBaseProperties, Item> {
   }
 
   _initScrollView(): void {
-    const scrollingEnabled = this.option('scrollingEnabled');
-    const pullRefreshEnabled = scrollingEnabled && this.option('pullRefreshEnabled');
+    const {
+      height,
+      width,
+      disabled,
+      showScrollbar,
+      useNativeScrolling,
+      bounceEnabled,
+      scrollByContent,
+      scrollByThumb,
+      pullingDownText,
+      pulledDownText,
+      refreshingText,
+      pageLoadingText,
+      scrollingEnabled,
+      pullRefreshEnabled,
+    } = this.option();
+
+    const isPullRefreshEnabled = scrollingEnabled && pullRefreshEnabled;
     const autoPagingEnabled = scrollingEnabled
       && this._scrollBottomMode()
       && !!this._dataController.getDataSource();
 
     this._scrollView = this._createComponent(this.$element(), getScrollView(), {
```

Comment:
The typing of ScrollView properties is incorrect here. I will explore how we can improve this